libs/documentation/src/lib/components/landing/link/link.module.ts

```diff
@@ -34,7 +34,7 @@ export const ROUTES = [
       title: 'Landing / Link',
       items: [
         {
-          pkg: 'layouts',
```

Comment:
It seems like quite a few of these files where package is layouts can be removed as well

Reply:
Removed un related demo's from the app